LiverMarkups/VTKWidgets/vtkSlicerBezierSurfaceRepresentation3D.cxx

```diff
@@ -581,8 +581,8 @@ void vtkSlicerBezierSurfaceRepresentation3D::CreateAndTransferVascularSegmentsTe
   this->VascularSegmentsTexture->SetWrapS(vtkTextureObject::ClampToBorder);
   this->VascularSegmentsTexture->SetWrapT(vtkTextureObject::ClampToBorder);
   this->VascularSegmentsTexture->SetWrapR(vtkTextureObject::ClampToBorder);
-  this->VascularSegmentsTexture->SetMinificationFilter(vtkTextureObject::Linear);
-  this->VascularSegmentsTexture->SetMagnificationFilter(vtkTextureObject::Linear);
+  this->VascularSegmentsTexture->SetMinificationFilter(vtkTextureObject::Nearest);
+  this->VascularSegmentsTexture->SetMagnificationFilter(vtkTextureObject::Nearest);
```

Comment:
How does this affect the resulting visualization?

Reply:
> How does this affect the resulting visualization?
Because the scalar of VascularSegmentsTexture is short int, so linear Magnification/Minification between two different labeled areas will result in a float number that hasn't been defined in the [shader](https://github.com/ALive-research/Slicer-Liver/blob/e5e68a936806b76283f80c0373aac75fa9e20b34/LiverMarkups/VTKWidgets/vtkOpenGLResection2DPolyDataMapper.cpp#L233-L273)

Reply:
Of course! I was still thinking on your latest advances with scalar volumes :)